### PR TITLE
add building, building_join, address, and enhanced_ownership schemas

### PR DESCRIPTION
**Pull request by vjcharles**
These were sourced from gdoc spreadsheets.
All the column names and values haven't been changed.

There are directories for each and the schema name is slightly redundant but may be nice having the schema type in the file name if they are used elsewhere.

I can add a verse schema too if we want that here.

I did notice we don't include short cut names or the category field to denote public or private for the other schemas.

related to https://github.com/loveland/wdwot/issues/5349

### CODE REVIEW

**Review by Blake-Loveland — APPROVED** [review]

**Comment by hampelm** [comment]
> The keys are identical to the spreadsheet versions including having spaces between words; do we want to have keys all lower-snakecase?

I would vote yes, lower snakecase, for consistency 

> There isn't the 'categories' array on each field like in our parcel schema. How should that be represented in these schema files?

I know it is inconsistent but with my desire for consistency, but I don't think these need `categories`. We use the `categories` to filter out data from different people. Or each could have a blank categories list if you want, or with just a "base" value or something if you would like that consistency. But I don't think it's necessary from my vantage point. 

> Do we want to include verse too? The FS doesn't need it but I can add it with these other schemas since I'm here. 

I would say pass for now to keep the scope down here but it is good to know we have a place to put it when we need it.

**Comment by vjcharles** [comment]
Thanks, Matt! The latest changes are up--snake_case and with an empty `categories` array.

The `verse_schema.yml` converted nicely and can be easily added any time.

File: address/address_schema.yml [hunk 1]
```diff
@@ -0,0 +1,326 @@
+---
+a_address:
+  sequence_number: '1'
+  first_seen: '1.0'
```
Comment (hampelm) [inline address/address_schema.yml:4]: I think these should probably be numbers, so they can be compared

Comment (vjcharles) [reply address/address_schema.yml:4]: Sounds good!

I have a question for you related to [wdwot#5409](https://github.com/loveland/wdwot/pull/5409) but it is relevant asking here too:
Currently the parcel schema endpoint exposes most of these columns. Based on our public schema documentation, I believe we can just return in json these columns:  field name, human name, description, type, examples and tier.

The other schemas have fewer columns but the basics are: field name, description, type and examples.

**Comment by quidquid** [comment]
Hrm, this won't stage until the schema PR is merged; the gh action clones the schema repo separately, not using the submodule ref

File: address/address_schema.yml (hunk 1 above)
Comment (vjcharles) [reply address/address_schema.yml]: Thanks! Can we merge this PR then? I'm a little tentative doing the merge here without an explicit thumbs up :)

**Review by hampelm — APPROVED** [review]

**Pull request merged**